Do not abort when the fips mode flag is simply inaccessible because of
insufficient permissions.
diff -up libgcrypt-1.4.4/src/fips.c.no-access libgcrypt-1.4.4/src/fips.c
--- libgcrypt-1.4.4/src/fips.c.no-access	2009-01-29 17:37:12.000000000 +0100
+++ libgcrypt-1.4.4/src/fips.c	2009-01-29 17:37:15.000000000 +0100
@@ -155,6 +155,7 @@ _gcry_initialize_fips_mode (int force)
         fclose (fp);
       }
     else if ((saved_errno = errno) != ENOENT
+             && saved_errno != EACCES
              && !access ("/proc/version", F_OK) )
       {
         /* Problem reading the fips file despite that we have the proc
